test/OpenSpaceToolkit/Astrodynamics/Trajectory/Orbit/Models/Propagated/STK_EGM96_70x70_2hr_run.csv

```diff
@@ -0,0 +1,122 @@
+YAM.UTCGregorian,YAM.ElapsedSecs,YAM.EarthICRF.X,YAM.EarthICRF.Y,YAM.EarthICRF.Z,YAM.EarthICRF.VX,YAM.EarthICRF.VY,YAM.EarthICRF.VZ

```

Comment:
Again, feels like this was generated by GMAT and not STK?

Reply:
see above